<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryChooseWidget</class>
 <widget class="QWidget" name="CategoryChooseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CategoryChooseListView" name="listView">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{
	background-color:#ffffff;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>13</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="KPosUpArrowButton" name="upArrowButton" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ededed;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPosDownArrowButton" name="downArrowButton" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CategoryChooseListView</class>
   <extends>QListView</extends>
   <header>CategoryChooseListView.h</header>
  </customwidget>
  <customwidget>
   <class>KPosUpArrowButton</class>
   <extends>QWidget</extends>
   <header>KPosArrowButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPosDownArrowButton</class>
   <extends>QWidget</extends>
   <header>KPosArrowButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
